src/pheval/cli_pheval_utils.py

```diff
@@ -11,19 +14,77 @@
     metavar="FILE",
     help="Path to the semantic similarity profile to be scrambled.",
 )
-def scramble_semsim():
+def scramble_semsim(input: click.Path):
     """scramble_semsim"""
     print("running pheval_utils::scramble_semsim command")
 
 
-@click.command()
+@click.command("scramble-phenopackets")
 @click.option(
-    "--input",
-    "-i",
+    "--phenopacket-dir",
+    "-P",
+    metavar="PATH",
+    required=True,
+    help="Path to phenopackets directory",
+)
+@click.option(
+    "--scramble-factor",
+    "-s",
+    metavar=float,
+    required=True,
+    default=0.5,
+    show_default=True,
+    help="Scramble factor for randomising phenopacket phenotypic profiles.",
+    type=float,
+)
+@click.option(
+    "--output-file-suffix",
+    "-o",
+    metavar="<str>",
+    required=True,
+    help="Suffix to append to output file",
+)
+@click.option(
+    "--output-dir",
+    "-O",
+    metavar="PATH",
+    required=True,
+    help="Path for creation of output directory",
+    default="noisy_phenopackets",
+)
+def scramble_phenopackets_command(
+    phenopacket_dir: click.Path,
+    scramble_factor: float,
+    output_file_suffix: str,
+    output_dir: click.Path,
+):
+    """Generate noisy phenopackets from existing ones."""
+    create_scrambled_phenopackets(output_dir, output_file_suffix, phenopacket_dir, scramble_factor)
+
+
+@click.command("semsim-heatmap")
+@click.option(
+    "--semsim1",
```

Comment:
lets standardise diff parameters to `left`, `right`, so `-L/--semsim-left`, `-R/--semsim-right`, (make sure -R and -L are not taken and not used as standard parameters), see https://github.com/monarch-initiative/operations/issues/118